users/clients/service.go

```diff
@@ -163,13 +163,23 @@ func (svc service) ListClients(ctx context.Context, token string, pm mfclients.P
 	if err != nil {
 		return mfclients.ClientsPage{}, err
 	}
+
 	if pm.SharedBy == MyKey {
 		pm.SharedBy = id
 	}
 	if pm.Owner == MyKey {
 		pm.Owner = id
 	}
 	pm.Action = "c_list"
+
+	// If the user is admin, fetch all things from database.
+	pol := policies.Policy{Subject: id, Object: clientsObjectKey, Actions: []string{listRelationKey}}
```

Comment:
Use `p` instead of `pol`. Does this mean that we disable filtering by, say, owner for admin?

Reply:
Currently we don't filter by `owner`. This is because visibility changes the owner field

Reply:
We need to provide that filter in the future. Please open a ticket for that.